### ISSUE TITLE
regression in 2016.3 from today?

### ISSUE THREAD

**Issue by sjorge** [saltstack/salt#31563]
### Expected Behavior
No warnings, also looks like file_ignore_regex does not get applied.

### Actual Behavior
```bash
[root@cronos /srv/salt]# salt cronos grains.items
[WARNING ] Key file_ignore_regex with value ['/\\.svn($|/)', '/\\.git($|/)'] has an invalid type of list, a bool is required for this value
[WARNING ] Key master_tops with value {} has an invalid type of dict, a bool is required for this value
[WARNING ] Key file_ignore_regex with value ['/\\.svn($|/)', '/\\.git($|/)'] has an invalid type of list, a bool is required for this value
[WARNING ] Key master_tops with value {} has an invalid type of dict, a bool is required for this value
...
```
### Steps to Reproduce Issue
Run 2016.3 build from feb 29th ~21:15 UTC 

### Versions Report
2016.3 build from feb 29th ~21:15 UTC

**Comment by jfindlay** [comment]
Thanks, @sjorge.

**Comment by sjorge** [comment]
#31723 fixes the file_ignore_regex in 2015.8 and would need forward merging to 2016.3 and develop. master_tops is already fixed in 2015.8 and pending forward-merge.

Alternatively #31707 fixes both in 2016.3, but previous one mention would probably be better to merge. I will keep both up for now.

**Comment by sjorge** [comment]
Fixed, closign
